composer.json

```diff
@@ -67,7 +71,8 @@
 
 		"symfony/browser-kit": "~2.7",
 		"silex/web-profiler": "~2.0",
-		"sorien/silex-dbal-profiler": "~2.0"
+		"sorien/silex-dbal-profiler": "~2.0",
+		"wmde/fundraising-frontend-content": "dev-master as 1.0.x-dev"
```

Comment:
AFAIK this is _the_ way to track master, otherwise we get

     Could not find package wmde/fundraising-frontend-content at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability

or we would have to reduce the minimum stability of the consuming project (which we don't want, I assume)